content/compose/gettingstarted.md

```diff
@@ -351,4 +351,4 @@ $ docker compose down --volumes
 - Try the [Sample apps with Compose](https://github.com/docker/awesome-compose)
 - [Explore the full list of Compose commands](reference/index.md)
 - [Explore the Compose file reference](compose-file/index.md)
-- To learn more about volumes and bind mounts, see [Manage data in Docker](../storage/index.md)
\ No newline at end of file
+- To learn more about volumes and bind mounts, see [Manage data in Docker](../storage/index.md)
```

Comment:
Not sure why this is counted as a diff, I haven't touched it and nothing seems to have changed